.github/workflows/_test_te.yaml

```diff
@@ -104,147 +104,73 @@ jobs:
             ${{ env.BADGE_FILENAME_FULL }}
 
   te-multi-gpu:
+    uses: ./.github/workflows/_test_slurm_pyxis.yaml
     strategy:
       matrix:
-        N_GPU: [1, 2]
+        N_GPU: [2, 4, 8]
       fail-fast: false
-    runs-on: ubuntu-22.04
+    secrets:
+      SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      SLURM_LOGIN_USER: ${{ secrets.CLUSTER_LOGIN_USER }}
+      CONTAINER_REGISTRY_TOKEN: ${{ secrets.github_token }}
+    with:
+      NAME: te-${{ matrix.N_GPU }}GPU
+      ARTIFACT_NAME: ${{ inputs.ARTIFACT_NAME }}-${{ matrix.N_GPU }}GPU
+      SLURM_LOGIN_HOSTNAME: ${{ vars.HOSTNAME_SLURM_LOGIN }}
+      SLURM_SCRATCH_PATH: /nfs/cluster
+      NODES: 1
+      GPUS_PER_NODE: ${{ matrix.N_GPU }}
+      NTASKS: 1
+      NTASKS_PER_NODE: 1
+      TIME_LIMIT: '00:10:00'
+      EXTRA_EXPORTS: 'VOCAB_PATH=gs://t5-data/vocabs/cc_all.32000.100extra/sentencepiece.model'
+      IMAGE: ghcr.io/nvidia/jax:upstream-pax
+      SRUN_PREAMBLE: |
+        nvidia-smi
+        pip install \
+          pytest \
+          pytest-reportlog \
+          cuda-python \
+          -r ${SRC_PATH_TE}/examples/jax/encoder/requirements.txt
+      SRUN_SCRIPT: |
+        set -ex
+        cd ${SRC_PATH_TE}/examples/jax/encoder
+        pytest --report-log=/output/pytest.jsonl \
```

Comment:
Makes sense. I'll revise accordingly.

Reply:
Actually, now that you've renamed both reports to `pytest-report.jsonl`, is there any issue with counting test results? In particular, [this line](https://github.com/NVIDIA/JAX-Toolbox/pull/626/files#diff-0ca60f47d99fba066bb85abe3d3922a77a48ba00de7ba1ffea5b98ab58bcb14cR161) will look for all `pytest-report.jsonl`, and likely by the time it runs, the unit tests would have finished and it appears the unit test artifact will also be downloaded.

Apologies I wasn't clear before, but I was thinking something like `pytest-report-unit.jsonl` for the unit tests and `pytest-report-slurm.jsonl` to differentiate.

Reply:
Yeah that's possible, but would likely be temporary, because I plan to update the TE unit tests to use the `_test_unit` workflow after this PR gets merged. This job that you are referencing should really only be generating the sitrep for the MGMN tests.

Reply:
I feel that it is redundant to have slurm/unit in the jsonl file names because the information will have already been conveyed in the artifact name.

Reply:
Got it, if you're going to update TE tests in another PR than my consideration is moot. Thanks for explaining